Fix plugin docker mount documentation (#3124)
neo4j/docs-operations: modules/ROOT/pages/docker/docker-compose-standalone.adoc

    image: neo4j:latest
    volumes:  # <1>
        - /$HOME/neo4j/logs:/logs
        - /$HOME/neo4j/conf:/conf
        - /$HOME/neo4j/data:/data
        - /$HOME/neo4j/plugins:/plugins # <2>
    environment:
        - NEO4J_AUTH=neo4j/your_password # <3>
    ports:
      - "7474:7474"
      - "7687:7687"
    restart: always
----
<1> Mount the _/$HOME/neo4j/<..>:_ directories to local directories on your host machine to store logs, configuration, data, and plugins.
For more information about mounting volumes, see xref:docker/mounting-volumes.adoc[].

> <2> To prevent broken plugin lookups or permission errors, add the strict configuration path mapping directly under the environment variables: `- NEO4J_server_directories_plugins=/plugins`.

Author: ```suggestion
<2> To prevent broken plugin lookups, keep the plugins volume mapping and add the matching configuration path under your environment variables: `- NEO4J_server_directories_plugins=/plugins`.
The configuration setting does not replace `/plugins` volume mount and only controls where Neo4j looks for plugins.
```

    image: neo4j:latest
    volumes:  # <1>
        - /$HOME/neo4j/logs:/logs
        - /$HOME/neo4j/conf:/conf
        - /$HOME/neo4j/data:/data
        - /$HOME/neo4j/plugins:/plugins # <2>
    environment:
        - NEO4J_AUTH=neo4j/your_password # <3>
    ports:
      - "7474:7474"
      - "7687:7687"
    restart: always
----
<1> Mount the _/$HOME/neo4j/<..>:_ directories to local directories on your host machine to store logs, configuration, data, and plugins.
For more information about mounting volumes, see xref:docker/mounting-volumes.adoc[].

> <2> To prevent broken plugin lookups, keep the plugins volume mapping and add the matching configuration path under your environment variables: `- NEO4J_server_directories_plugins=/plugins`.

Author: ```suggestion
<2> If you face broken plugin lookups, keep the plugins volume mapping and add the matching configuration path under your environment variables: `- NEO4J_server_directories_plugins=/plugins`.
```

Author: I don't want to make it sound like this is a mandatory step

Reviewer: It's not, and also setting the `NEO4J_server_directories_plugins` variable will do nothing for permissions problems.

        - /$HOME/neo4j/logs:/logs
        - /$HOME/neo4j/conf:/conf
        - /$HOME/neo4j/data:/data
        - /$HOME/neo4j/plugins:/plugins # <2>
    environment:
        - NEO4J_AUTH=neo4j/your_password # <3>
    ports:
      - "7474:7474"
      - "7687:7687"
    restart: always
----
<1> Mount the _/$HOME/neo4j/<..>:_ directories to local directories on your host machine to store logs, configuration, data, and plugins.
For more information about mounting volumes, see xref:docker/mounting-volumes.adoc[].
<2> If you face broken plugin lookups, keep the plugins volume mapping and add the matching configuration path under your environment variables: `- NEO4J_server_directories_plugins=/plugins`.

> The configuration setting does not replace `/plugins` volume mount and only controls where Neo4j looks for plugins.

Reviewer: No we don't need this. The original issue is unreproducible and this step is 100% not necessary.
